frontend/src/search.js

```diff
@@ -85,7 +85,7 @@ search.addWidgets([
   },
   panel({
     templates: {
-      header: 'Location',
+      // header: 'Location',
```

Comment:
Remove the `panel` wrappers altogether, we don't need them anymore.

Reply:
Removed the panel wrappers.